107/src/main/java/org/ehcache/jsr107/ConfigurationMerger.java

```diff
@@ -162,6 +164,43 @@
     return builder;
   }
 
+  private static <K, V> CacheConfigurationBuilder<K, V> addDefaultCopiers(CacheConfigurationBuilder<K, V> builder, Class keyType, Class valueType ) {
```

Comment:
Why not having this logic in `DefaultCopyProvider` so that ehcache API users can also benefit from those copiers?